packages/story-editor/src/components/publishModal/test/content.js

```diff
@@ -35,7 +35,7 @@ const mockInputValues = {
   [INPUT_KEYS.SLUG]: '2001-space-odyssey',
 };
 
-describe('publishModal/mainContent', () => {
+describe('publishModal/content', () => {
```

Comment:
We should rename the import on line 29 `import Content from '../content';`

Reply:
And rename Component on line 71  `MainContent` to `Content`

Reply:
ah yup, i totally missed this file. thank you!